### PR TITLE
Create gem skeleton & dependencies

### PR DESCRIPTION
**Pull request by cilim**
This pushes the gem skeleton and our RuboCop configuration file from the [Default Rails Template](https://github.com/infinum/default_rails_template)

### CODE REVIEW

**Comment by cilim** [comment]
@d4be4st do we need any kind of licence or code of conduct? I see that [Phrasing's got one](https://github.com/infinum/phrasing/blob/master/MIT-LICENSE)

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+inherit_gem:
```
Comment (d4be4st) [inline .rubocop.yml]: this cant be here I think as it will do nothing. this needs to be in project/.rubocop.yml.

**Review by d4be4st — CHANGES_REQUESTED** [review]
Please add a documentation how to use this gem

File: .rubocop.yml (hunk 1 above)
Comment (cilim) [reply .rubocop.yml]: Probably...I thought to test this out once we release the gem, cause neither of `rubocop-gems` that I've tried locally (by providing `path` in the Gemfile) worked.

[rubocop-rspec](https://github.com/rubocop-hq/rubocop-rspec/blob/master/.rubocop.yml) has a weird config where it requires its own gem name

**Review by d4be4st — APPROVED** [review]

**Pull request merged**